stdlib/_io.pyi

```diff
@@ -173,19 +166,15 @@ class TextIOWrapper(TextIOBase, _TextIOBase, TextIO, Generic[_BufferT_co]):  # t
         line_buffering: bool | None = None,
         write_through: bool | None = None,
     ) -> None: ...
-    # These are inherited from TextIOBase, but must exist in the stub to satisfy mypy.
-    def __enter__(self) -> Self: ...
-    def __iter__(self) -> Iterator[str]: ...  # type: ignore[override]
-    def __next__(self) -> str: ...  # type: ignore[override]
-    def writelines(self, lines: Iterable[str], /) -> None: ...  # type: ignore[override]
     def readline(self, size: int = -1, /) -> str: ...  # type: ignore[override]
-    def readlines(self, hint: int = -1, /) -> list[str]: ...  # type: ignore[override]
     # Equals the "buffer" argument passed in to the constructor.
     def detach(self) -> _BufferT_co: ...  # type: ignore[override]
     # TextIOWrapper's version of seek only supports a limited subset of
     # operations.
     def seek(self, cookie: int, whence: int = 0, /) -> int: ...
 
+# The C implementation is not a subclass of TextIOWrapper at runtime,
```

Comment:
The C implementation is the one that virtually all users would use, right? In that case, it would be better to follow the C implementation.

Reply:
I wouldn't mind untangling that, but I think that's for another MR. I can remove the note if you'd prefer, it's not terribly important to capture that here right now.